src/Presentation/Pagination/Types/TPPagination.cs

```diff
@@ -0,0 +1,17 @@
+namespace Presentation.Pagination.Types
+{
+    using System.Diagnostics.CodeAnalysis;
+    using Business.Pagination;
+    using global::GraphQL.Types;
+    using Style;
+
+    [SuppressMessage(Category.Default, Check.CA1724, Justification = Reason.Readability)]
+    public sealed class TPPagination : ObjectGraphType<PPagination>
+    {
+        public TPPagination()
+        {
+            Field(pagination => pagination.PageSize).Description("This is the page size");
```

Comment:
Should we also add the Name and Description for this type ?

Reply:
done